### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rubyllamadora**
## Motivation and Context

**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-

## Changed

1.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [x] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

**Review by tim-yao — CHANGES_REQUESTED** [review]
Hi @rubyllamadora as discussed, need to add template update and update script as well. Thanks!

File: .circleci/config.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@ jobs:
 
   audit:
     docker:
-      - image: cypress/browsers:node12.0.0-chrome73
+      - image: cypress/browsers:node12.19.0-chrome86-ff82
```
Comment (rubyllamadora) [inline .circleci/config.yml:96]: I have updated to later version as the new eslint update requires atleast node >= 12.13.0

File: .circleci/config.yml [hunk 2]
```diff
@@ -267,21 +267,21 @@ workflows:
           filters:
             branches:
               only:
-                - /^release\/.*/
+                - /^(release|feature)\/.*/
```
Comment (rubyllamadora) [inline .circleci/config.yml:270]:
```suggestion
                - /^release\/.*/
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -267,21 +267,21 @@ workflows:
...
       - e2e-example:
           requires:
             - build
           filters:
             branches:
               only:
-                - /^release\/.*/
+                - /^(release|feature)\/.*/
```
Comment (rubyllamadora) [inline .circleci/config.yml:277]:
```suggestion
                - /^release\/.*/
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -267,21 +267,21 @@ workflows:
...
       - e2e-vue-example-app:
           requires:
             - build
           filters:
             branches:
               only:
-                - /^release\/.*/
+                - /^(release|feature)\/.*/
```
Comment (rubyllamadora) [inline .circleci/config.yml:284]:
```suggestion
                - /^release\/.*/
```

File: .circleci/config.yml (hunk 2 above)
Comment (rubyllamadora) [reply .circleci/config.yml:270]: I just added this to make sure dependency updates have all e2e tests working as expected.

File: .eslintrc.js [hunk 5]
```diff
@@ -1,33 +1,55 @@
 // https://eslint.org/docs/user-guide/configuring
-
 module.exports = {
   root: true,
   parserOptions: {
-    parser: 'babel-eslint',
+    parser: '@babel/eslint-parser',
```
Comment (rubyllamadora) [inline .eslintrc.js:5]: ESLint no longer used `babel-eslint` as they will no longer be maintained in the future. https://babel.dev/blog/2020/07/13/the-state-of-babel-eslint

File: packages/create-ripple-app/generators/updates/1.31.0/saofile.js [hunk 6]
```diff
@@ -14,7 +14,8 @@ module.exports = {
           data.dependencies['nuxt'] = `2.14.10`
           data.dependencies['@nuxtjs/gtm'] = '^2.4.0'
           data.dependencies['run-script-os'] = `^1.1.3`
-          data.devDependencies['babel-eslint'] = `^10.1.0`
+          data.devDependencies['@babel/eslint-parser'] = `^^7.12.1`
+          data.devDependencies['@babel/eslint-plugin'] = `^^7.12.1`
```
Comment (rubyllamadora) [inline packages/create-ripple-app/generators/updates/1.31.0/saofile.js:18]: I should revert this and create another file `1.32.0`

File: packages/ripple-nuxt-tide/test/unit/__snapshots__/markup-transpiler.test.js.snap [hunk 7]
```diff
@@ -3,12 +3,12 @@
 exports[`Markup transpiler should able transpile html into vue template with plugins 1`] = `
 "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
 <rpl-text-link :url=\\"linkData0.href\\" theme=\\"primary\\" :text=\\"linkData0.text\\"></rpl-text-link>
-<h4>Sort out your&apos;&gt;&lt;&amp;&apos;&quot;s finances</h4>
+<h4>Sort out your'&gt;&lt;&amp;'\\"s finances</h4>
```
Comment (rubyllamadora) [inline packages/ripple-nuxt-tide/test/unit/__snapshots__/markup-transpiler.test.js.snap:6]: This is from the eslint upgrade. I need to review what causes the change and if this is the desired change.

**Review by rubyllamadora — COMMENTED** [review]
Hey @tim-yao, this is already using axios 0.21.1 however it is causing error in the API, specifically in axios.js `$axios.onError`. I need to review if we will rollback to axios 0.19.0 or move forward fixing the breaking change.

Oh wait, it's working in https://app-ripple-pr-871.lagoon.vicsdp.amazee.io/
It doesn't work in my local env :(

File: packages/create-ripple-app/generators/updates/1.31.0/saofile.js (hunk 6 above)
Comment (tim-yao) [reply packages/create-ripple-app/generators/updates/1.31.0/saofile.js:18]: yes, please

Comment (tim-yao) [reply packages/create-ripple-app/generators/updates/1.31.0/saofile.js:18]: I think we also need to remove `babel-eslint` from the project package.json

File: .circleci/config.yml (hunk 2 above)
Comment (tim-yao) [reply .circleci/config.yml:270]: I think we can use a different prefix for dependency update task e.g `dep-update`?

Comment (rubyllamadora) [reply .circleci/config.yml:270]: Agree with dep-update. Also add `renovate/` here if we are doing it in the future?

**Comment by rubyllamadora** [comment]
I'll rename this @tim-yao to `dep-update` so it can run e2e tests.

**Pull request closed without merging**